doc/articles/features/pointers-keyboard-and-other-user-inputs.md

```diff
@@ -0,0 +1,183 @@
+  have the `IsRightButtonPressed` set (in addition of the `IsBarrelButtonPressed`). On WinUI and Android you get this flag only if the barrel 
+  button was pressed at the moment where you touched the screen, otherwise you will have the `IsLeftButtonPressed` and the `IsBarrelButtonPressed`.
+* For pen and fingers, the `Holding` event is not raised after a given delay like on WinUI, 
+  but instead we rely on the fact that we usually get a lot of moves for those kind of pointers, 
+  so we raise the event only when we get a move that exceed defined thresholds for holding.
+* On WASM, Shapes must have a non null Fill to receive pointer events (setting the Stroke is not sufficient).
+* On WASM if the user scrolls in diagonal (e.g. with a Touchpad), but you mark as `Handled` pointer events only for vertical scrolling,
+  then the events for the horizontal scroll component won't bubble through the parents.
+
+### Pointer capture
+
+The capture of pointer is handled in managed code only. On WebAssembly Uno however still requests the browser to capture the pointer,
+but Uno does not rely on native `[got|lost]pointercapture` events.
+
+### iPadOS mouse support
+
+To differentiate between mouse and touch device type for pointer events, include the following in your app's `Info.plist`:
+
+```
+<key>UIApplicationSupportsIndirectInputEvents</key>
+<true/>
+```
+
+Without this key the current version of iPadOS reports mouse interaction as normal touch.
+
+## Manipulation Events
+
+They are generated from the PointerXXX events (using the `Windows.UI.Input.GestureRecognizer`) and are bubbling in managed only.
+
+Currently there is no intertia support, so the `IsInertial` will always be `false` and the `UIElement.ManipulationInertiaStarting` event 
+will never be fired. The `Velocities` properties of event args are not implemented neither.
+
+## Gesture Events
+
+They are generated from the PointerXXX events (using the `Windows.UI.Input.GestureRecognizer`) and are bubbling in managed only.
+
+Note that `Tapped` and `DoubleTapped` are not linked in any way to a native equivalent, but are fully interpreted in managed code.
+
+In order to match the WinUI behavior, on WASM the default "Context menu" of the browser is disabled (except for the `TextBox`), 
+no matter if you use / handle the `RightTapped` event or not.
+Be aware that on some browser (Firefox), user can still request to get the "Context menu" on right click.
+
+## Drag and drop
+
+Those events are also 100% managed events, built from the PointerXXX events (using the `Windows.UI.Input.GestureRecognizer`)
+
+### Intra-app drag and drop support
+
+A _drag and drop_ operation can be used to move content within an app, but it can also be used to **copy** / **move** / **link** between apps.
+While inter-app _drag and drop_ is supported on all platforms without limiations, intra-app _drag and drop_ requires platfrom specific support.
```

Comment:
```suggestion
While inter-app _drag and drop_ is supported on all platforms without limitations, intra-app _drag and drop_ requires platform specific support.
```

I corrected the typos but... is it true? You write "supported on all platforms without limitations" but there's a lot of 'No' below :p

Reply:
Yeah it's true: the "inter-app" does not suffer real limitation, just a minor bug which won't have any functional impact.

Reply:
Ok then are you mixing up 'inter-' and 'intra-'?

Reply:
Whoups rigth! ... Fixed